pkg/apis/core/validation/validation.go

```diff
@@ -2609,6 +2610,42 @@ func validatePullPolicy(policy core.PullPolicy, fldPath *field.Path) field.Error
 	return allErrors
 }
 
+var supportedResizeResources = sets.NewString(string(core.ResourceCPU), string(core.ResourceMemory))
+var supportedResizePolicies = sets.NewString(string(core.NoRestart), string(core.RestartContainer))
+
+func validateResizePolicy(policyList []core.ResizePolicy, fldPath *field.Path) field.ErrorList {
+	allErrors := field.ErrorList{}
+
+	if !utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) && len(policyList) > 0 {
```

Comment:
generally we do not want to check gates in validation - that happens before we ever get here.  See DropDisabledPodFields()

Reply:
thanks! i'll fix this.

Reply:
fixed in commit https://github.com/vinaykul/kubernetes/pull/1/commits/5126b9e1e93d2530ac820981c93986d64e46faa3